src/lib/charts/dataFromResources.ts

```diff
@@ -1,6 +1,15 @@
 import { createDateTimes } from '../table/tableData'
-import { Series } from '../timeseries/timeSeries'
-import { SeriesArrayData, TimeSeriesData } from '../timeseries/types/SeriesData'
+import type { Series } from '../timeseries/timeSeries'
+import type {
+  SeriesArrayData,
+  SeriesData,
+  TimeSeriesData,
+} from '../timeseries/types/SeriesData'
+
+export function isReliableData(data: SeriesArrayData | SeriesData) {
+  const unreliableFlags = ['6', '7', '8']

```

Comment:
This should probably be in the configuration? Maybe hard-coded for now but from the back-end at some point?